content/en/docs/tasks/debug-application-cluster/debug-service.md

```diff
@@ -18,10 +18,10 @@ you to figure out what's going wrong.
 ## Running commands in a Pod
 
 For many steps here you will want to see what a Pod running in the cluster
-sees.  The simplest way to do this is to run an interactive alpine Pod:
+sees.  The simplest way to do this is to run an interactive minibusy Pod:
 
 ```none
-kubectl run -it --rm --restart=Never alpine --image=alpine sh
+kubectl run -it --rm --restart=Never minibusy --image=radial/busyboxplus:curl sh
```

Comment:
I'd like to make the first use of “minibusy” link to a page about minibusy, but I can't easily find one on the web.

That's a concern! @ljnaresh, why is switching to minibusy a good idea?

Reply:
@sftim I have used the image `radial/busyboxplus:curl` as it was the proposed solution for the issue #21436. 

Additionally, I could see that it provides cleaner output  with `nslookup` without the message "server can't find xxx: NXDOMAIN"  seen with `alpine` image. 

I see the concern with "minibusy", how about changing pod name to busyboxplus.

Reply:
> I see the concern with "minibusy", how about changing pod name to busyboxplus.

Sure, either `busyboxplus` or `busyboxplus-curl` would be fine.

Reply:
I have replaced the image with `gcr.io/google-containers/busybox` and corrected the pod name.